package.json

```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "start": "NODE_ENV=production node ./src/server/server.babel.js",
     "dev": "NODE_ENV=development node ./src/server/server.babel.js",
-    "build": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\"",
+    "build:themes": "babel-node ./webpack/utils/buildThemeJSON.js",
```

Comment:
I'm not sure I follow 100% on how the theme is building. Wasn’t it a matter of calculating the values on the fly in dev and building an object in production? Or am I totally lost?